docs/gke/iap.md

```diff
@@ -162,18 +162,74 @@ gcloud projects add-iam-policy-binding $PROJECT \
   --member user:${USER_EMAIL}
 ```
 
-### Self signed certificates and browser security warnings
+### Obtaining a valid certificate from LetsEncrypt
 
 Since you are using a self signed certificate chrome and other browsers will give you a warning like
 
 ```
 Attackers might be trying to steal your information from ${ENDPOINT}(for example, passwords, messages, or credit cards). Learn more
 NET::ERR_CERT_AUTHORITY_INVALID
 ```
-  * You will need to ignore these warnings
-  * To avoid these warnings you will need to use a certificate signed by a signing authority
-  * [Lets Encrypt](https://letsencrypt.org/) and other sites provide free signed certificates.
 
+We can use [cert-manager](https://github.com/jetstack/cert-manager) to automatically obtain and refresh LetsEncrypt certificates.
```

Comment:
Could you add a link or explanation for LetsEncrypt

Reply:
Done